pkgs/development/tools/frink/default.nix

```
@@ -27,9 +27,18 @@ stdenv.mkDerivation rec {
 
     cp ${src} $out/lib/frink.jar
 
+    # Generate rlwrap helper files.
+    # See https://frinklang.org/fsp/colorize.fsp?f=listUnits.frink
+    # and https://frinklang.org/fsp/colorize.fsp?f=listFunctions.frink
+    ${jdk}/bin/java -classpath "$out/lib/frink.jar" frink.gui.FrinkStarter -e 'joinln[lexicalSort[units[]]]' > $out/lib/unitnames.txt
```

```suggestion
    java -classpath "$out/lib/frink.jar" frink.gui.FrinkStarter -e 'joinln[lexicalSort[units[]]]' > $out/lib/unitnames.txt
```

With `jdk` put in `nativeBuildInputs` additionally.

---

Done (and squashed), thanks.